src/sparseml/optim/manager.py

```diff
@@ -520,6 +527,31 @@ def qat_active(self, epoch: float) -> bool:
             else False
         )
 
+    def _info_log_metadata(self):
+        logging.info(
```

Comment:
nit: create a logger object `_LOGGER` like other files in repo and switch to `_LOGGER.info`